api/__tests__/app.test.js

```diff
@@ -81,16 +81,65 @@ describe("/api", () => {
         });
     });
   });
-
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("should return the specified article comments", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const { comments } = res.body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments).toHaveLength(11);
+          comments.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id");
```

Comment:
I would suggest just checking the property exist is insufficient, can we also check the data type of each key's value